tests/AssetManagerTest/View/Helper/AssetTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace AssetManagerTest\View\Helper;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use AssetManager\View\Helper\Asset;
+use Zend\ServiceManager\ServiceManager;
+
+class AssetTest extends TestCase
+{
+    public function testInvoke()
+    {
+        /* TODO need help
```

Comment:
How can I instantiate a ServiceManager (comprehensive all configs) to pass to the view helper?